java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -3187,11 +3209,33 @@ public final ColumnVector matchesRe(String pattern) {
    * @param pattern Regex pattern to match to each string.
    * @return New ColumnVector of boolean results for each string.
    */
+  @Deprecated
   public final ColumnVector containsRe(String pattern) {
+    return containsRe(new RegexProgram(pattern, CaptureGroups.NON_CAPTURE));
+  }
+
+  /**
+   * Returns a boolean ColumnVector identifying rows which
+   * match the given RegexProgram pattern starting at any location.
+   *
+   * ```
+   * cv = ["abc","123","def456"]
+   * result = cv.contains_re("\\d+")
```

Comment:
This example still uses the deprecated method, needs to be updated to show how to use the new API.